.github/workflows/continuous-integration.yml

```diff
@@ -155,3 +155,43 @@ jobs:
         set PYTHONPATH=%PYTHONPATH%;%GITHUB_WORKSPACE%/bin/%KRATOS_BUILD_TYPE%
         set PATH=%PATH%;%GITHUB_WORKSPACE%/bin/%KRATOS_BUILD_TYPE%/libs
         python kratos/python_scripts/run_tests.py -l small -c python
+
+
+  centos:
+    runs-on: centos-latest
```

Comment:
github doesn't provide Centos.
See here: https://docs.github.com/en/actions/reference/specifications-for-github-hosted-runners#supported-runners-and-hardware-resources

=> thats why we need the Docker container